TicTacToe.test/UnitTests.fs

```diff
@@ -365,8 +365,8 @@ let canComputerFindDiagonalWins () =
 
 [<Fact>]
 let canStupidComputerMakeMove () =    
-    let board = {BoardSize = 3; CurrentBoard = [None;None;None;None;None;None;None;None;None]; TurnNumber = 1; IsInverted = false}
+    let board = {BoardSize = 3; CurrentBoard = [Some('O');Some('O');None;None;Some('X');None;None;None;None]; TurnNumber = 1; IsInverted = false}
     let player1 = {PlayerType = 1; ComputerAlgorithm = minimaxStarter; PlayerCharacter = 'O'}
     let player2 = {PlayerType = 1; ComputerAlgorithm = minimaxStarter; PlayerCharacter = 'X'}
     let testMove = stupidComputerMove(board,player1,player2) 
-    Assert.True(testMove > -1 && testMove < 9)
\ No newline at end of file
+    Assert.True(testMove > 1 && testMove < 9 && not(testMove = 4))
```

Comment:
It made sense to make it test a particular board.  Before, I was just testing a blank board, but that did not prove that the changes that were made actually worked.  In the test board, the spaces 0, 1, and 4 are already selected.  That is why it should not pick 4, and why the "testMove > -1" changed to "testMove > 1"